docs/faq.md

```diff
@@ -0,0 +1,89 @@
+jupytext --to ipynb *.md                        # convert all .md files to notebooks with no outputs
+jupytext --to ipynb --execute *.md              # convert all .md files to notebooks and execute them
+jupytext --set-formats ipynb,md --execute *.md  # convert all .md files to paired notebooks and execute them
+```
+
+## Which files should I version control?
+
+Unless you want to version control the output cells, you should version the text file only (and add `*.ipynb` to `.gitignore`). As discussed above, Jupyter will let you open the text representation as a notebook and will re-create the `.ipynb` file when you save the notebook.
+
+Note that if you version control both the `.md` and `.ipynb` files, you can configure `git diff` to [ignore the diffs on the `.ipynb` files](https://github.com/mwouts/jupytext/issues/251).
+
+## Jupyter warns me that the file has changed on disk
+
+By default, Jupyter saves your notebook every 2 minutes. Fortunately, it is also aware that you have edited the text file.
+
+You can deactivate the autosave in Jupyter with the Jupytext Menu in Jupyter Notebook, and with the _Autosave Document_ setting in JupyterLab. See [here](https://stackoverflow.com/questions/25631344/turn-off-autosave-in-ipython-notebook/56549758#56549758) if you want to permanently deactivate the autosave in Jupyter Notebook.
+
+## When I refresh, Jupyter warns me that my notebook has unsaved changes
+
+Oh - you have edited both the notebook and the paired text file at the same time? Backup the text file (`git stash`), save the notebook, and merge your changes on the text file (`git stash pop`). When you're done, refresh the notebook in Jupyter.
+
+If your IDE has the ability to compare the changes in memory versus on disk (like PyCharm), you can simply save the notebook and let your IDE do the merge.
+
+## Jupyter complains that the `.ipynb` file is more recent than the text representation
+
+This happens if you have edited the `.ipynb` file outside of Jupyter. Manual action is requested as the paired text representation may be outdated. Please edit (`touch`) the paired `.md` or `.py` file if it is not outdated, or if it is, delete it, or update it with
+```bash
+jupytext --sync notebook.ipynb
+```
+
+## Can I use Jupytext with Jupyter Hub, Binder, Nteract, Colab, Saturn or Azure?
+
+Jupytext is compatible with Jupyter Hub (execute `pip install jupytext --user` to install it in user mode) and with Binder (add `jupytext` to the project requirements and `jupyter lab build` to `postBuild`).
+
+If you use another editor than Jupyter Notebook, Lab or Hub, you probably can't get Jupytext there. However you can still use Jupytext at the command line to manually sync the two representations of the notebook:
+
+```bash
+jupytext --set-formats ipynb,py:light notebook.ipynb   # Pair a notebook to a light script
+jupytext --sync notebook.ipynb                         # Sync the two representations
+```
+
+## If only I had known of Jupytext before!
```

Comment:
I like the personality coming across in this one, but I think it'd still benefit from a more clear idea of what the section covers...e.g., "Can I re-write my git history to use text files instead of notebooks?"